start-cb.sh

```diff
@@ -221,6 +222,7 @@ start_uaa() {
 
     debug "waits for uaadb get registered in consul"
     wait_for_service uaadb
+    sleep 20
```

Comment:
Postgres container needs time to come up but wait_for_service checking just the consul registration